internal/utils/flags/flags.go

```diff
@@ -95,6 +108,11 @@ type BoolFlag struct {
 
 // Register registers the boolean flag with the provided flag set
 func (f BoolFlag) Register(fs *pflag.FlagSet) {
+	if f.Deprecator != nil {
+		f.Deprecator.Deprecate(fs, f.Name)
```

Comment:
[question] what happens if we didn't `return` here (and/or had the `Deprecate` call at the end of the `Register` impl)?

would that then start displaying the deprecation message if/when `--include-dependencies` popped up?

Reply:
I did the Deprecation call at the end of this function, and it cannot work, after `setNormalFunc` is called this flag becomes a no-op(the value func its trying to call is `cmd.inputs.IncludeDependencies`), the implementation of `setNormalfunc` does replace old name with new name, but it does it this way `f.actual[nname] = flag`, the result of this is, all attribute of the flag does not change except the name, so the behavior is, new flag will point to the deprecated func.  So I choose to do it this way because the `AddFlag` function checks the normalized name first, that ensures we have the correct behavior here.